blockly-svelte/rollup.config.js

```diff
@@ -0,0 +1,50 @@
+import svelte from 'rollup-plugin-svelte';
```

Comment:
Please add copyright header. You can find it in any of the other files in this repository.

Reply:
added